DEFCON Z/Assets/Scripts/UI/SettingsMenu.cs

```diff
@@ -18,22 +45,67 @@ public void SetDifficulty(int difficulty)
         {
             switch (difficulty)
             {
-                case 1:
+                case 0:
                     Debug.Log("Difficulty set to Easy");
                     Difficulty.SelectedDifficulty = Difficulty.Easy;
                     
                     break;
-                case 2:
+                case 1:
                     Debug.Log("Difficulty set to Normal");
                     Difficulty.SelectedDifficulty = Difficulty.Normal;
                     break;
-                case 3:
+                case 2:
                     Debug.Log("Difficulty set to Hard");
                     Difficulty.SelectedDifficulty = Difficulty.Hard;
                     break;
                 default:
                     break;
             }
         }
+
+        /// <summary>
+        /// Swaps the active state of the provided game object, and the game object this script is attached to
+        /// </summary>
+        /// <param name="other"></param>
+        public void ToggleActive(GameObject other)
```

Comment:
I believe this method is doing too much and can be confusing. A better alternatives will be creating an extension method that toggles boolean.